pkg/controller/migplan/validation.go

```diff
@@ -381,7 +345,8 @@ func (r ReconcileMigPlan) validateSourceCluster(plan *migapi.MigPlan) error {
 			Type:     SourceClusterNotReady,
 			Status:   True,
 			Category: Critical,
```

Comment:
Not related to this PR but `Reason` key is missing

Reply:
Added the `Reason` key.